packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss

```diff
@@ -15,19 +15,26 @@
     align-items: center;
     display: flex;
     flex: 0 0 auto;
-    padding-left: var(--terra-collapsible-menu-view-menu-button-padding-left, 0.714rem);
+    padding-left: var(--terra-collapsible-menu-view-menu-button-padding-left, 0.5rem);
 
     &.hidden {
       display: none;
       width: 0;
     }
 
     .menu-button-icon {
-      background: var(--terra-collapsible-menu-view-menu-button-icon-background, inline-svg('<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#1c1f21" d="M24 18.6a5.4 5.4 0 1 1-5.4 5.4 5.38 5.38 0 0 1 5.4-5.4zm18.6 0a5.4 5.4 0 1 1-5.4 5.4 5.38 5.38 0 0 1 5.4-5.4zm-37.2 0A5.4 5.4 0 1 1 0 24a5.38 5.38 0 0 1 5.4-5.4z"/></svg>'));
-      background-position: center 0.15em;
+      background: var(--terra-collapsible-menu-view-menu-button-icon-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#1c1f21" d="M18.6 24c0-3 2.4-5.4 5.4-5.4s5.4 2.4 5.4 5.4-2.4 5.4-5.4 5.4c-3 0-5.4-2.4-5.4-5.4zm0-18.6C18.6 2.4 21 0 24 0c3 0 5.4 2.4 5.4 5.4 0 3-2.4 5.4-5.4 5.4-3 0-5.4-2.4-5.4-5.4zm0 37.2c0-3 2.4-5.4 5.4-5.4s5.4 2.4 5.4 5.4c0 3-2.4 5.4-5.4 5.4-3 0-5.4-2.4-5.4-5.4z"/></svg>'));
+      background-position: center;
       background-repeat: no-repeat;
-      background-size: cover;
+      background-size: auto;
       display: inline-block;
+      position: relative;
+      top: var(--terra-collapsible-menu-view-menu-button-icon-top, 0);
+      vertical-align: var(--terra-collapsible-menu-view-menu-button-icon-vertical-align, -0.14285rem);
+      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
Should we be adding vender prefixes to the styles?

Reply:
That is needed for IE specific SVG/icon and text alignment details we spent many hours on here:  [#2523](https://github.com/cerner/terra-core/pull/2523/files#diff-b2ae2172c829371a523555d2e0626b06)

Reply:
Can we add some inline comments regarding this so future engineers/unfamiliar people know why we've added these?

Reply:
Added comment https://github.com/cerner/terra-framework/pull/966/commits/739d850974e32564a29991f569a06ae34445548e